tests/ui/attributes/mixed_export_name_and_no_mangle.rs

```diff
@@ -0,0 +1,10 @@
+// issue: rust-lang/rust#47446
+
```

Comment:
we should check that the suggestion is application and indeed fix the issue
```suggestion
//@ run-rustfix
```